core/storage/question/gae_models.py

```diff
@@ -100,3 +100,40 @@ def create(
             language_code=language_code)
 
         return question_model_instance
+
+
+class QuestionSkillLinkModel(base_models.VersionedModel):
```

Comment:
This shouldn't be a versioned model, it should subclass from BaseModel instead. There's no need to revert or maintain a history since the only property is a difficulty int which is easy to change on the fly.

Sorry for not catching this earlier.

Reply:
Done